api/.env-template

```diff
@@ -0,0 +1,27 @@
+# This is the port the API will be available at
```

Comment:
@marcorichetta how's this for a single env template instead?

Reply:
Don't forget to change the filename in the readme.

https://github.com/adamblanchard/hyf-project-template/blob/7312ecaf12fc20ff82afc265edcc0ea71e05cf1e/api/README.md?plain=1#L13